### PR TITLE
https://issues.redhat.com/browse/ACM-10400--versions and other update…

### PR DESCRIPTION
**Pull request by swopebe**
…s to older content based on developer feedback

### CODE REVIEW

File: clusters/hosted_control_planes/disable_auto_import.adoc [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+Any hosted clusters that were previously imported are not affected, even if you disable automatic import. When you upgrade to {mce-short} 2.5 and automatic import is enabled, all hosted clusters that are not imported are automatically imported if their control planes are available.
+
+*Note:* All {product-title-short} add-ons are also enabled if {product-title-short} is installed. 
+
+When automatic import is disabled, only newly created hosted clusters are not automatically imported. Hosted clusters that were already imported are not affected. You can still manually import clusters by using the console or by creating the `ManagedCluster` and `KlusterletAddonConfig` custom resources. 
 
 To disable the automatic import of hosted clusters, complete the following steps:
```
Comment (swopebe) [inline clusters/hosted_control_planes/disable_auto_import.adoc:13]: @rokej please look at this one? This was me removing 2.4 and then reviewing the content.

File: clusters/hosted_control_planes/monitor_user_workload_disconnected.adoc [hunk 2]
```diff
@@ -35,7 +35,7 @@ data:
 [#verify-hosted-control-plane-feature-disconnected-aws]
 == Verifying the status of the hosted control plane feature 
 
-Starting with {mce-short} 2.4, the hosted control plane feature is enabled by default. 
+The hosted control plane feature is enabled by default. 
 
 . If the feature is disabled and you want to enable it, enter the following command. Replace `multiclusterengine` with the name of your {mce-short} instance:
```
Comment (swopebe) [inline clusters/hosted_control_planes/monitor_user_workload_disconnected.adoc:41]: @rokej and this one.

File: install/upgrade_disconnected.adoc [hunk 3]
```diff
@@ -5,13 +5,9 @@ See the steps and information to upgrade {product-title} in a disconnected netwo
```
Comment (swopebe) [inline install/upgrade_disconnected.adoc:5]: @dislbenn can you review this topic clean -up and attempt to make more general--or pass it to someone who has some time?

File: clusters/hosted_control_planes/disable_auto_import.adoc (hunk 1 above)
Comment (rokej) [reply clusters/hosted_control_planes/disable_auto_import.adoc:13]: This looks good. Thank you.

**Review by rokej — APPROVED** [review]

**Review by dislbenn — APPROVED** [review]

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**